clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -388,7 +434,7 @@ public boolean poll(Timer timer) {
 
         if (subscriptions.partitionsAutoAssigned()) {
             if (protocol == null) {
-                throw new IllegalStateException("User confingure ConsumerConfig#PARTITION_ASSIGNMENT_STRATEGY_CONFIG to empty " +
+                throw new IllegalStateException("User configured ConsumerConfig#PARTITION_ASSIGNMENT_STRATEGY_CONFIG to empty " +
```

Comment:
While we're at it, could we change this message to use the actual property rather than a pointer to the code?

Reply:
Good point!